src/mctp-bench.c

```diff
@@ -683,7 +683,12 @@ static int command(mctp_eid_t eid, int net, enum command command,
 	req->magic = COMMAND_MAGIC;
 	req->version = COMMAND_VERSION;
 	req->command = command;
-	getrandom(&req->iid, sizeof(req->iid), 0);
+	ssize_t rand_len = getrandom(&req->iid, sizeof(req->iid), 0);
+	if (rand_len < 0) {
+		err(EXIT_FAILURE, "getrandom");
+	} else if (rand_len != sizeof(req->iid)) {
+		errx(EXIT_FAILURE, "getrandom is insufficiently random");
+	}
```

Comment:
It shouldn't be possible to fail for <= 256 length buffer on anything since really ancient kernels.